app/controllers/dashboard/payments_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Dashboard
+  class PaymentsController < AuthenticatedController
+    content_security_policy false
```

Comment:
A content security policy is something that allows you to whitelist the domains from which you will accept resources like web fonts, iframes, and Javascript files. They are enforced by the browser, and Firefox's implementation of this enforcement is broken. Since we're loading offsite resources (Javascript from Stripe), I'm playing it safe and disabling the default policy on this page only.